### PR TITLE
update opencart 2.0

### PR DESCRIPTION
**Pull request by candrasaputra**

### CODE REVIEW

File: opencart2.0.x-2.2.x/upload/catalog/controller/payment/xendit.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 class Controllerpaymentxendit extends Controller
 {
-    const EXT_ID_PREFIX = 'xendit_opencart_';
+    const EXT_ID_PREFIX = 'xendit-opencart-';
```
Comment (hakiemaul) [inline opencart2.0.x-2.2.x/upload/catalog/controller/payment/xendit.php]: we go with `opencart-xendit-` yaa

File: opencart2.0.x-2.2.x/upload/catalog/controller/payment/xenditcc.php [hunk 2]
```diff
@@ -2,7 +2,10 @@
 
 require_once(DIR_SYSTEM . 'library/xendit.php');
 
-class Controllerpaymentxenditcc extends Controller {
+class Controllerpaymentxenditcc extends Controller
+{
+    const EXT_ID_PREFIX = 'xendit-opencart-';
```
Comment (hakiemaul) [inline opencart2.0.x-2.2.x/upload/catalog/controller/payment/xenditcc.php]: same as above

File: opencart2.0.x-2.2.x/upload/catalog/controller/payment/xenditcc.php [hunk 3]
```diff
@@ -145,6 +151,14 @@ public function process_3ds() {
                     'store_name' => $store_name
                 )
             );
+            
+            if ($charge['status'] !== 'CAPTURED') {
```
Comment (hakiemaul) [inline opencart2.0.x-2.2.x/upload/catalog/controller/payment/xenditcc.php]: the original logic lives inside `process_order` might want to merge it there instead

**Review by hakiemaul — APPROVED** [review]

File: opencart2.0.x-2.2.x/upload/catalog/controller/payment/xendit.php [hunk 4]
```diff
@@ -168,6 +182,7 @@ private function process_order($response, $order_id)
             $this->response->setOutput($message);
         } else {
             $message = 'Invoice not paid or settled. Cancelling order. Charge id: ' . $response['id'];
```
Comment (IreneGohtami) [inline opencart2.0.x-2.2.x/upload/catalog/controller/payment/xendit.php]: Please change text to: 'Invoice not paid or settled. Cancelling order. **Invoice ID**: '

**Review by IreneGohtami — APPROVED** [review]

**Pull request merged**